controllers/vrg_volrep.go

```diff
@@ -479,7 +502,13 @@ func (v *VRGInstance) PVUploadToObjectStore(s3ProfileName string, pvc *corev1.Pe
 			pvc.Name, s3ProfileName, err)
 	}
 
-	if err := UploadPV(objectStore, v.s3KeyPrefix(), pv.Name, pv); err != nil {
+	return v.UploadPVAndPVCtoS3(s3ProfileName, objectStore, &pv, pvc)
```

Comment:
(future) Investigate if this be done as a single upload to maintain as a transaction, instead of 2 uploads to S3. That would help ensure that the pair is always found, and in extreme cases it is not that the PV is found but not the PVC.

Reply:
Yeah, I was thinking of that. We'll address it. An issue was created here: https://github.com/RamenDR/ramen/issues/832